modules/nw-dual-stack-convert.adoc

```diff
@@ -2,7 +2,12 @@
 [id="nw-dual-stack-convert_{context}"]
 = Converting to a dual-stack cluster network
 
-As a cluster administrator, you can convert your single-stack cluster network to a dual-stack cluster network. After converting to a dual-stack network, new and existing pods have dual-stack networking enabled.
+As a cluster administrator, you can convert your single-stack cluster network to a dual-stack cluster network. 
+
+[IMPORTANT]
+====
+After converting your cluster to use dual-stack networking, only newly created pods are assigned IPv6 addresses. For any pods created before the conversion, you must re-create the pods for them to receive IPv6 addresses.
```

Comment:
Just a nitpick suggestion:
```suggestion
After converting your cluster to use dual-stack networking, only new pods are assigned IPv6 addresses. For any pods created before the conversion, you must re-create the pods for them to receive IPv6 addresses.
```

Reply:
Thanks. Updated!

Reply:
Or you could take it even further, something in this direction:
```suggestion
After converting your cluster to use dual-stack networking, 
you must re-create any existing pods that you wish to receive IPv6 addresses 
because only new pods are assigned IPv6 addresses.
```

Reply:
Updated.